core/data-plane/data-plane-framework/src/main/java/org/eclipse/dataspaceconnector/dataplane/framework/pipeline/PipelineServiceImpl.java

```diff
@@ -106,6 +115,7 @@ public CompletableFuture<StatusResult<Void>> transfer(DataSink sink, DataFlowReq
             return noSourceFactory(request);
         }
         var source = sourceFactory.createSource(request);
+        monitor.debug(() -> format("transferring from %s to %s.", source, sink));
```

Comment:
Can you capitalize the emessage?

Reply:
I updated the messages. Thanks, @jimmarino.